src/monsters.h

```diff
@@ -97,7 +97,11 @@ struct BestiaryInfo
 	std::string className = "";
 	uint8_t difficulty = 0;
 	uint8_t occurrence = 0;
+	std::tuple<uint32_t, uint32_t, uint32_t> totalKills;
```

Comment:
Do you see any advantages on using a tuple vs three separate `uint32_t`s?

Reply:
I don't think it has any advantage other than looking prettier 😊